config/locales/en.yml

```diff
@@ -78,29 +78,39 @@ en:
       confirm_withdraw: Withdraw your application
   root:
     index:
-      title: "Connecting hearts and homes"
-      worldwide: "worldwide"
-      description: "A free application for pet foster and adoption organizations who want to save time and manage their operations in one place."
-      easy_to_use: "Easy to use"
-      no_download: "No dowload needed"
-      manage: "Manage operations in one place"
-      save: "Save time and spend it on what matters"
-      btn_title: "Request an account"
-      subtitle: "Manage your operations"
-      subtitle_highlight: "in one place"
-      subtitle_description: "No more spreadsheets, no more paper, no more headaches."
+      title: "Manage"
+      pet: "Pets"
+      description: "Bring your operations to the digital age. Handle everything from staffing, foster scheduling, and adoptions, all from one free app."
```

Comment:
could we change the word `app` to `website`? After seeing this on my screen, alive, I don't want to give people the impression this is a mobile app. So can we change this for all cases below as well?

Reply:
sure. Will replace this as mention